### ISSUE TITLE
Not possible to choose device tracker in person list

### ISSUE THREAD

**Issue by pszafer** [home-assistant/home-assistant-polymer#4699]
## Checklist

- [x] I have updated to the latest available Home Assistant version.
- [x] I have cleared the cache of my browser.
- [x] I have tried a different browser to see if it is related to my browser.

## The problem
Problem only occurs in Chrome/Chromium based browsers. No Firefox.

If I choosed Persons list and try to connect person with device tracker I cannot choose device_tracker by mouse cursor, only keyboard.
Tried in incognito mode, same behaviour.
No issues in Firefox.

![hass_person](https://user-images.githubusercontent.com/690510/73524677-e7a78300-440d-11ea-946e-4b56b5cfa9b6.gif)

## Expected behavior
Possiblity to choose device tracker.

## Steps to reproduce
Go to Person list, choose Person and try to choose device_tracker connected to person.

## Environment

**No issues in Firefox**

- Home Assistant release with the issue: 0.104.3
- Last working Home Assistant release (if known): 0.102 I think.
- UI Type (States or Lovelace): Lovelace
- Browser and browser version: Chromium 79.0.3945.117 (Official version) Arch Linux (64-bits)
- Operating system: Arch

## Javascript errors shown in your browser console/inspector
No console errors/warnings.

**Comment by iantrich** [comment]
I am unable to replicate on 0.105.0b

**Comment by pszafer** [comment]
I think I will close it for now. It is something with my browser, but don't know what...
I deleted my profile and it is present even on fresh dev environment.

If somebody will have similar issue and more ability to provide useful info what's going on then it can be reopened.
I use this browser: https://aur.archlinux.org/packages/chromium-vaapi/
I tried to disable Hardware acceleration, but it doesn't help.
